### PR TITLE
Add php-cs-fixer to github actions

### PR DESCRIPTION
**Pull request by wgevaert**
Let's hope builds do trigger this time.
See [this issue](https://github.com/WikibaseSolutions/php-cypher-dsl/issues/45) and [this pr that is exactly the same](https://github.com/WikibaseSolutions/php-cypher-dsl/pull/44).

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -55,8 +56,14 @@ jobs:
       - name: Install dependencies
         run: composer update --working-dir ${{ env.INSTALL_PATH }}
 
+      - name: Run php-cs-fixer on src
+        run: ./vendor/bin/php-cs-fixer fix --dry-run --verbose --rules=${{ env.FIXER_RULES }} ./src
```
Comment (marijnvanwezel) [inline .github/workflows/main.yml:60]: Use `composer exec php-cs-fixer` instead of `./vendor/bin/php-cs-fixer`.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -55,8 +56,14 @@ jobs:
...
+
+      - name: Run php-cs-fixer on tests
+        run: ./vendor/bin/php-cs-fixer fix --dry-run --verbose --rules=${{ env.FIXER_RULES }} ./tests
```
Comment (marijnvanwezel) [inline .github/workflows/main.yml:63]: Same as above.

**Review by marijnvanwezel — CHANGES_REQUESTED** [review]

**Comment by marijnvanwezel** [comment]
The pipeline is not being run because of the following error:

```
Invalid workflow file: .github/workflows/main.yml#L30
You have an error in your yaml syntax on line 30
```

See also [here](https://github.com/WikibaseSolutions/php-cypher-dsl/actions/runs/2257145811).

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -27,6 +27,7 @@ jobs:
       COMPOSER_VERSION: 2
       INSTALL_PATH: php-cypher-dsl
       REPOSITORY_NAME: php-cypher-dsl
+      FIXER_RULES: @PSR12,align_multiline_comment,array_indentation,blank_line_before_statement,indentation_type,no_trailing_whitespace,no_unused_imports,no_useless_else,no_useless_return,ordered_imports,phpdoc_scalar,return_assignment,simplified_if_return,single_line_after_imports,switch_case_space,trailing_comma_in_multiline,ternary_operator_spaces
```
Comment (marijnvanwezel) [inline .github/workflows/main.yml:30]: Use quotes here to fix the YAML syntax error.

Comment (wgevaert) [reply .github/workflows/main.yml]: tnx!

**Review by marijnvanwezel — APPROVED** [review]

**Pull request merged**